Formula/libmms.rb

```diff
@@ -20,7 +21,7 @@ class Libmms < Formula
   depends_on "glib"
 
   def install
-    ENV.append "LDFLAGS", "-liconv"
+    on_macos { ENV.append "LDFLAGS", "-liconv" }
```

Comment:
Doesn't this mean it `uses_from_macos`?

Reply:
`iconv` should be in `glibc` on Linux, so no extra dependency.

macOS just needs the extra flag to find library.